src/main/java/com/coveros/selenified/element/Assert.java

```diff
@@ -156,17 +156,15 @@ boolean isPresentTable(String expected) {
     String[] getAttributes(String attribute, String expected) {
         // wait for the element
         if (!isPresent()) {
-            return null; // NOSONAR - returning an empty array could be confused
-            // with no attributes
+            return null; // returning an empty array could be confused with no attributes
         }
         file.recordExpected(EXPECTED + element.prettyOutput() + " " + expected + " attribute <b>" + attribute + "</b>");
         // check our attributes
         Map<String, String> attributes = element.get().allAttributes();
         if (attributes == null) {
             file.recordActual("Unable to assess the attributes of " + element.prettyOutputEnd(), Success.FAIL);
             file.addError();
-            return null; // NOSONAR - returning an empty array could be confused
-            // with no attributes
+            return null; // returning an empty array could be confused with no attributes
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Return an empty array instead of null. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1168)